### PR TITLE
Add the novelist

### PR DESCRIPTION
**Pull request by lestrrat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,4 +182,11 @@ User files a bug,
 Twitter, posts to Reddit, asks on Stackoverflow, emails support,
 emails sales, privately DMs some core developers ....
 
+## The novelist
+
+User files a bug, maintainers asks for minimal reproduction test.
+User does _NOT_ provide test case, instead opts to write out an entire short story on what he belives is happening, starting at his childhood, a mysterious problem that he encountered and the wonderous half-human, half-lizard being that helped his way through trying to fix the problem, a story full of allegory, but zero code.
+
+Maintainers still have no idea what the bug really is, because they don't understand what the user did
+
```
Comment (bradfitz) [inline README.md:191]: You cut off the word STOP from the previous entry.

File: README.md [hunk 2]
```diff
@@ -182,4 +182,11 @@ User files a bug,
 Twitter, posts to Reddit, asks on Stackoverflow, emails support,
 emails sales, privately DMs some core developers ....
 
+## The novelist
+
+User files a bug, maintainers asks for minimal reproduction test.
+User does _NOT_ provide test case, instead opts to write out an entire short story on what he belives is happening, starting at his childhood, a mysterious problem that he encountered and the wonderous half-human, half-lizard being that helped his way through trying to fix the problem, a story full of allegory, but zero code.
+
+Maintainers still have no idea what the bug really is, because they don't understand what the user did
```
Comment (bradfitz) [inline README.md:190]: Missing final period.

File: README.md [hunk 3]
```diff
@@ -182,4 +182,11 @@ User files a bug,
 Twitter, posts to Reddit, asks on Stackoverflow, emails support,
 emails sales, privately DMs some core developers ....
 
+## The novelist
```
Comment (bradfitz) [inline README.md:185]: Nice one :)

File: README.md (hunk 1 above)
Comment (lestrrat) [reply README.md]: Oh, I thought it was "STOP (all of the above)". reinstated it.

**Comment by lestrrat** [comment]
@bradfitz done :D

**Comment by ljharb** [comment]
This is akin to those recipe websites where you have to read an anthology before getting to the instructions.

**Pull request merged**